CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.4
+* Update database mappings based on common-schema change [#286](https://github.com/microsoft/common-schema/pull/286).
+* Update http dependency name based on common-schema change [#285](https://github.com/microsoft/common-schema/pull/285).
```

Comment:
let's detail these changes since they are user facing (and users don't know what common-schema is)